index.js

```diff
@@ -45,7 +45,16 @@ function Choo (opts) {
   // properties that are part of the API
   this.router = nanorouter({ curry: true })
   this.emitter = nanobus('choo.emit')
-  this.state = { events: this._events }
+
+  var events = { events: this._events }
+  if (this.hasWindow) {
+    this.state = window.initialState
+      ? xtend(window.initialState, events)
```

Comment:
Offtopic. Why are we using `xtend` instead of `Object.assign`?

Reply:
Browser compat — switching to `Object.assign()` would be a semver major. Agree we should for v7 tho :v: